code/modules/hallucinations/effects/blind_rush_hallucination.dm

```diff
@@ -0,0 +1,69 @@
+/**
+  * # Hallucination - Blind Rush
+  *
+  * Makes target blind and causes them to see semi-transparent humanoids running at them.
+  */
+/datum/hallucination_manager/blind_rush
+	initial_hallucination = /obj/effect/hallucination/no_delete/blind_rusher
+	trigger_time = 34 //total length of the hallucination is a little more than ten seconds
+
+
+/datum/hallucination_manager/get_spawn_location()
```

Comment:
```suggestion
/datum/hallucination_manager/blind_rush/get_spawn_location()
```
You're overwriting procs for all hallucinations currently.

Reply:
Thank you, all overwritten proc should have been changed correctly now!